tasks/travis-pr-title-lint.js

```diff
@@ -18,25 +18,53 @@ var writeCommitMessagesDocURL =
   '/CONTRIBUTING.md#writing-commit-messages';
 
 module.exports = function(grunt) {
-  function countGitMergeCommits() {
+  function findMergeBase() {
     return new Promise(function(resolve, reject) {
-      exec('git rev-list --count HEAD ^master', function(err, stdout) {
+      exec('git merge-base HEAD master', function(err, stdout) {
         if (err) {
           reject(err);
         } else {
-          resolve(parseInt(stdout));
+          var baseCommit = stdout.trim();
+          if (process.env.VERBOSE === 'true') {
```

Comment:
I think that being able to produce a more verbose output based on an environment var it could be helpful if we have more issues with this checks in the future, and so I'm probably going to leave this verbose logging in the final version.